js/components/report/iframe-answer.js

```diff
@@ -12,13 +13,8 @@ export default class IframeAnswer extends PureComponent {
     this.toggleIframe = this.toggleIframe.bind(this);
   }
 
-  toggleIframe(event) {
-    const { question } = this.props;
-    if (question.get("displayInIframe")) {
-      // If displayInIframe == true, we won't follow the link.
-      event.preventDefault();
-      this.setState({iframeVisible: !this.state.iframeVisible});
-    }
```

Comment:
I'm pretty certain that LabBooks have a reporting_url, but they should be shown in an iframe. I remember we talked about this, so probably this is now handled elsewhere. I just wanted to flag it, incase it was missed.

Reply:
Yup. LabBooks have `displayInIframe`, which is respected. The refactoring removed a useless href in the toggle link for embedded iframes, and only calls `toggleIframe` for embedded iframes (including LabBooks), so this check was no longer needed.